Dockerfiles/Dockerfile.package

```diff
@@ -1,5 +1,14 @@
 FROM golang:1.10
 
+ARG GPG_PUB_KEY_FILE
+ARG GPG_SEC_KEY_FILE
+ARG GPG_RPM_USER
+ARG GPG_RPM_EMAIL
+ARG WORKER_RPM
+ARG BCDA_RPM
+ARG BCDA_GPG_RPM_PASSPHRASE
+ARG GPG_PASSWORD_FILE
+
```

Comment:
Do you need all of those environment variables?  It looks like you are only passing in `BCDA_GPG_RPM_PASSPHRASE`.  All the other ones can probably be removed, right?